Nos/Service/FileStorage/FileStorageAPIClient.swift

```diff
@@ -87,16 +99,24 @@ class NostrBuildAPIClient: FileStorageAPIClient {
         }
     }
 
-    /// Creates a URLRequest and Data to be uploaded to the file storage API.
+    /// Creates a URLRequest and Data from a file URL to be uploaded to the file storage API.
     func uploadRequest(fileAt fileURL: URL, apiURL: URL) throws -> (URLRequest, Data) {
+        assert(fileURL.isFileURL, "The URL must point to a file.")
+        return try uploadRequest(
+            data: try Data(contentsOf: fileURL),
+            filename: fileURL.lastPathComponent,
+            apiURL: apiURL
+        )
+    }
+    
+    /// Creates a URLRequest and Data to be uploaded to the file storage API.
+    private func uploadRequest(data: Data, filename: String, apiURL: URL) throws -> (URLRequest, Data) {
         var request = URLRequest(url: apiURL)
         request.httpMethod = "POST"
```

Comment:
Maybe this is fine as-is, but maybe I should have used the enum above when I originally wrote this:

```suggestion
        request.httpMethod = HTTPMethod.post.rawValue
```

It's not a big deal either way -- just something I noticed while reviewing this.